src/components/HomePage/HothDescription.js

```diff
@@ -109,6 +116,23 @@ export default function HothDescription() {
 						}}/>
 					</Grid>
 				</Grid>
+				<Grid container spacing={isSmall ? 3 : 8} alignItems='center' justify='center'>
+					<Grid item xs={12} sm={8} md={6}>
+						<Img fluid={data.image3.childImageSharp.fluid} width='100%' style={{
```

Comment:
Similarly to `LeftGridItem` and `RightGridItem`, we can make abstract this `<Img>` component into another variable for better readability. You can pass in the image data as props.